hr_attendance_modification_tracking/migrations/12.0.1.2.0/pre-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Creu Blanca
```

Comment:
I think folder name should be '12.0.1.1.0', because the time_changed_manually field was added in that version.

Reply:
The version should be the destination version where it was implemented, so I think this is fine